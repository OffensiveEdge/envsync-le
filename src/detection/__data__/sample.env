# Production Environment Configuration
DATABASE_URL=postgresql://user:pass@localhost:5432/prod_db
API_KEY=sk-1234567890abcdef
DEBUG=false
PORT=8080
TIMEOUT=30
MAX_CONNECTIONS=100

# Feature Flags
ENABLE_CACHE=true
ENABLE_LOGGING=true
ENABLE_METRICS=false

# External Services
REDIS_URL=redis://localhost:6379
ELASTICSEARCH_URL=http://localhost:9200
S3_BUCKET=my-app-prod

# Security
JWT_SECRET=super-secret-jwt-key-here
ENCRYPTION_KEY=32-character-encryption-key
CORS_ORIGINS=https://app.example.com,https://admin.example.com

# Monitoring
SENTRY_DSN=https://key@sentry.io/project
LOG_LEVEL=info
METRICS_PORT=9090
