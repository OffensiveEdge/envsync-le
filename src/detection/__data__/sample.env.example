# Environment Configuration Template
# Copy this file to .env and fill in your values

# Database Configuration
DATABASE_URL=postgresql://user:password@localhost:5432/database_name
API_KEY=your-api-key-here
DEBUG=false
PORT=8080
TIMEOUT=30
MAX_CONNECTIONS=100

# Feature Flags
ENABLE_CACHE=true
ENABLE_LOGGING=true
ENABLE_METRICS=false

# External Services
REDIS_URL=redis://localhost:6379
ELASTICSEARCH_URL=http://localhost:9200
S3_BUCKET=your-s3-bucket-name

# Security (generate secure keys for production)
JWT_SECRET=your-jwt-secret-key-here
ENCRYPTION_KEY=your-32-character-encryption-key
CORS_ORIGINS=https://your-domain.com

# Monitoring
SENTRY_DSN=your-sentry-dsn-here
LOG_LEVEL=info
METRICS_PORT=9090
