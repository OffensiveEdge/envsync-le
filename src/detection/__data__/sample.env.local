# Local Development Overrides
DATABASE_URL=postgresql://user:pass@localhost:5432/dev_db
API_KEY=sk-dev-key-12345
DEBUG=true
PORT=3000
TIMEOUT=60

# Local Feature Flags
ENABLE_CACHE=false
ENABLE_LOGGING=true
ENABLE_METRICS=true

# Local Services
REDIS_URL=redis://localhost:6379
ELASTICSEARCH_URL=http://localhost:9200
S3_BUCKET=my-app-dev

# Local Security (less secure for dev)
JWT_SECRET=dev-jwt-secret-key
ENCRYPTION_KEY=dev-encryption-key-32-chars
CORS_ORIGINS=http://localhost:3000,http://localhost:3001

# Local Monitoring
SENTRY_DSN=
LOG_LEVEL=debug
METRICS_PORT=9091

# Additional local-only keys
LOCAL_DEV_MODE=true
MOCK_EXTERNAL_APIS=true
