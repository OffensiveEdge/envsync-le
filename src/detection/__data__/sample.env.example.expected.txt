DATABASE_URL
API_KEY
DEBUG
PORT
TIMEOUT
MAX_CONNECTIONS
ENABLE_CACHE
ENABLE_LOGGING
ENABLE_METRICS
REDIS_URL
ELASTICSEARCH_URL
S3_BUCKET
JWT_SECRET
ENCRYPTION_KEY
CORS_ORIGINS
SENTRY_DSN
LOG_LEVEL
METRICS_PORT
